### ISSUE TITLE
`createGlobalStyle` typing error: TemplateStringsArray

### ISSUE THREAD

**Issue by jimryanzulueta** [styled-components/styled-components#4055]
## System:
 - OS: macOS 13.0
 - CPU: (10) arm64 Apple M1 Pro
 - Memory: 281.03 MB / 32.00 GB
 - Shell: 5.8.1 - /bin/zsh
## Binaries:
 - Node: 16.15.0 - ~/Library/Caches/fnm_multishells/48343_1670904342698/bin/node
 - npm: 8.5.5 - ~/Library/Caches/fnm_multishells/48343_1670904342698/bin/npm
## npmPackages:
 - styled-components: ^6.0.0-rc.6 => 6.0.0-rc.6

## Environment

`npx envinfo --system --binaries --npmPackages styled-components,babel-plugin-styled-components --markdown`

## Reproduction

https://codesandbox.io/p/sandbox/pedantic-einstein-g6n337

I want to create a global style using `createGlobalStyle` but it returns this type error:
```
Type '{}' is missing the following properties from type 'TemplateStringsArray': raw, length, concat, join, and 23 more.ts(2740)

```
## Steps to reproduce
1. create any global style using `createGlobalStyle`
2. apply it to a component.
3. global style will show a type error.

## Expected Behavior
Global style is applied with no type errors.

## Actual Behavior
Global style is applied with a type error for the Global style.

**Comment by wadehammes** [comment]
Experiencing this also with latest RC

**Comment by sucodelarangela** [comment]
Having the same issue.

**Comment by jimryanzulueta** [comment]
Closing this. The newly released stable 6.0.0 have fixed this issue. 😀
